Dev: Add BuildIfChanged Task and Docs (#2202)
PowerShell/PowerShellEditorServices: README.md

Install-Module platyPS -Scope CurrentUser
```

> ### 4. Adjust `Nuget.Config` if necessary

Reviewer: ```suggestion
### 4. Adjust `nuget.config` if necessary
```

> Our NuGet configuration uses a secure feed with whitelisted third party dependency packages. If your contribution requires any changes to the included NuGet packages, you must disable this secure feed.

Reviewer: ```suggestion
Our NuGet configuration uses a secure feed with allow-listed third party dependency packages. If your contribution requires any changes to the included NuGet packages, you must disable this secure feed.
```
I know it's silly but our pipeline will flag us for "whitelist" and "blacklist" it has to be "allow-list" and "deny-list" 😆

Author: Not at all! Force of habit I probably need to change anyways.

First, run this command to prevent accidentally commiting changes to this file

```powershell

> git update-index --skip-worktree nuget.config

Reviewer: 👍

```

Then, either delete the file or remove the `packagesources` section to use nuget.org again. Your PR _will_ fail automated build checks and you _must_ inform us at the top of your PR so the appropriate packages can be added if approved.

## Build PowerShell Editor Services
Now you're ready to build the code. You can do so in one of two ways:

### PowerShell

```powershell

> PS C:\path\to\PowerShellEditorServices> Invoke-Build Build

Reviewer: ```suggestion
PS C:\src\PowerShellEditorServices> Invoke-Build
```

Might as well use the default task.

Author: The default task runs tests as well, wasn't sure if that was preferred.
